internal/services/monitor/monitor_diagnostic_setting_resource_test.go

```diff
@@ -249,6 +265,14 @@ func TestAccMonitorDiagnosticSetting_enabledLogs(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_monitor_diagnostic_setting", "test")
 	r := MonitorDiagnosticSettingResource{}
 	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.eventhub(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("enabled_log.#").HasValue("0"),
+			),
+		},
+		data.ImportStep(),
```

Comment:
we can remove this here since we are already testing this in `TestAccMonitorDiagnosticSetting_enabledLogsMix`

Reply:
removed